.github/ISSUE_TEMPLATE.md

```diff
@@ -1,3 +1,9 @@
+<!--
+WARNING: Please do not file issues for CocoaPods with projects that use React Native or Flutter. Please remove React Native or Flutter in your sample app in order
+to demonstrate properly an issue with CocoaPods itself. The integration between CocoaPods and React Native or Flutter makes it very difficult for us to diagnose
+and fix issues. Issues with React Native or Flutter will be closed.
```

Comment:
May want to add a suggestion to file the issues with those project, perhaps with links?

Reply:
yes!